schemas/2019-09-16/Microsoft.HealthcareApis.json

```diff
@@ -63,6 +63,24 @@
           "type": "string",
           "description": "An etag associated with the resource, used for optimistic concurrency when editing it."
         },
+        "identity": {
+            "type": "object",
+            "description": "Setting indicating whether the service has a managed identity associated with it.",
+            "properties": {
```

Comment:
Missing template expression definitions. See https://github.com/Azure/azure-resource-manager-schemas/blob/b74d7fc92aa09f138448da61ac54e8eca7fb765a/schemas/2019-08-01-preview/Microsoft.OperationalInsights.json#L16-L26 for an example of how this is declared

Reply:
Updated per the comments to add a definition for the identity type